container/container_test.go

```diff
@@ -153,7 +154,7 @@ func Test_ConfigGeneration(t *testing.T) {
 		image := "foo/foo:1.0.0"
 		command := []string{"date"}
 
-		cpus := float64(0.5)
+		cpus := float64(0.5) * float64(runtime.NumCPU())
```

Comment:
Had to make these relative to number of CPUs in the system so that we don't screw up Mesos accounting.